RUNNING.md

```diff
@@ -42,3 +42,78 @@ Another option is to get the game files by installing Caesar 3 using [WINE](http
 1. Run Julius with the path where the game is installed:
 
         $ julius path-to-c3-directory
+
+## Vita
+
+1. Install the .vpk like any other homebrew.
```

Comment:
I don't think that we can assume that everyone knows about vita jailbreaking and installing third-party application. Maybe we should give some pointers?

Reply:
Thanks for the feedback, I added a link to instructions for Vita and Switch.